Adding codesandboxes to play with the api and the project

PaulRosset opened the pull request:
[Codesandbox](codesandbox.io) is nice to create an easy and quick work environment to test and play with a project.

So I've made 3 separates "code sandboxes" that allow the user to play with the player more interactively without the overhead of setting a project from scratch locally and looking for content to play.

- [Playing a clear dash content](https://codesandbox.io/s/rx-player-classic-wc38j)
- [Playing an encrypted dash content](https://codesandbox.io/s/rx-player-drm-g51hb)
- [Playing a metaplaylist content](https://codesandbox.io/s/rx-player-metaplaylist-l0t0d)

More boxes could be added and for now, they are only inside the readme file but I'm open to discussion whether or not to add them elsewhere or just got rid of it.

peaBerberian commented:
Nice.
It is perfect there in the README.md IMO.

It feel weird to me though to have the instruction for `npm install` (btw we could also add a yarn command example, I'm now thinking about it) and even more the tutorial before this though. Or not...
What do you think? Do you want to move/change things around or is that a perfect place to you?

PaulRosset commented:
I removed the additional heading that I added earlier and added the link inside the existing block where we redirect users to the tutorials.

I find it nice to have those there as it has a direct link with what we explain above (The API).

peaBerberian commented:
Nice for both changes.

The last commit is to squash?

PaulRosset commented:
I squashed the whole :) thanks
